### ISSUE TITLE
Can warp differentiable with cloth materials?

### ISSUE THREAD

**Issue by fishfishson** [NVIDIA/warp#34]
Material parameters of mesh like mu lamba are all floats type. Can I replace them with wp array and set them requires_grad?

**Comment by cadop** [comment]
Maybe you could put this in discussion? Since its not an issue?

**Comment by mmacklin** [comment]
Hi @fishfishson, in the latest release we moved the triangle elastic materials to an array so you can now get gradients through them (https://github.com/NVIDIA/warp/blob/main/warp/sim/model.py#L1306).

Cheers,
Miles

**Comment by fishfishson** [comment]
> Hi @fishfishson, in the latest release we moved the triangle elastic materials to an array so you can now get gradients through them (https://github.com/NVIDIA/warp/blob/main/warp/sim/model.py#L1306).
> 
> Cheers, Miles

Thx for updating!
